.github/workflows/nessie.yaml

```diff
@@ -182,7 +183,7 @@ jobs:
 
       - name: Package Spark App
         working-directory: test/spark/app
-        run: sbt package
+        run: sbt ++2.11.12 package
```

Comment:
Does this work?
```suggestion
        run: sbt proj24/package
```

Reply:
++2.11.12 is ignored.
going with `sbt sonnets-246/package`